src/connect-to-browser.ts

```diff
@@ -1,32 +1,32 @@
 import * as childProcess from 'child_process';
 import * as path from 'path';
 import { promises as fs } from 'fs';
-import envPaths from 'env-paths';
 import * as puppeteer from 'puppeteer';
 // @ts-expect-error the bundle: syntax is from a plugin in the rollup config and TS does not know about it
 import startDisownedBrowserPath from 'bundle:./start-disowned-browser';
+import { fileURLToPath } from 'url';
 
-const readConfig = async (configPath: string) => {
+const readCache = async (cachePath: string) => {
```

Comment:
Renamed all the things config -> cache because cache is more descriptive of what it actually is. I think at one point I had planned to put other stuff than browser cache URL's in it, but not anymore.